features/fixtures/mazerunner/cxx-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXAnrScenario.java

```diff
@@ -0,0 +1,28 @@
+package com.bugsnag.android.mazerunner.scenarios;
+
+import com.bugsnag.android.Configuration;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+public class CXXAnrScenario extends Scenario {
+
+    static {
+        System.loadLibrary("monochrome");
```

Comment:
This can be removed - it's only necessary to load the `cxx-scenarios` SO library since the minimal test fixture has been added.